libraries/AC_AttitudeControl/AC_AttitudeControl_Multi.cpp

```diff
@@ -317,6 +317,20 @@ const AP_Param::GroupInfo AC_AttitudeControl_Multi::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("THR_G_BOOST", 7, AC_AttitudeControl_Multi, _throttle_gain_boost, 0.0f),
 
+    // @Param: PD_SCALE
+    // @DisplayName: PD scale factor
+    // @Description: Scale factor that is to be applied to PDs in PD_SCALE_AXES
+    // @Range: 0.1 0.9
+    // @User: Advanced
+    AP_GROUPINFO("PD_SCALE", 8, AC_AttitudeControl_Multi, _pd_scale_default, 1.0f),
+
+    // @Param: PD_SCL_AXES
+    // @DisplayName: PD scale axes
+    // @Description: Scale factor that is to be applied to PIDs in PD_SCL_AXES
+    // @Range: 0.1 0.9
```

Comment:
Roll/pitch/yaw bitmask?

Reply:
Yeah, I was also thinking an alternative would be to have a separate scale factor for each of RPY - which do you prefer?

Reply:
Separate values is only one more parameter and is much more flexible.

Reply:
Fixed